index.html

```diff
@@ -161,6 +161,41 @@ <h1>The Registration Process</h1>
 the modification request adheres to the following rules:
     </p>
 
+    <p>
+      The JSON representation of a DID Document is the same document (byte for byte equivalent) 
+      as the JSON-LD representation of a DID Document. 
+      JSON representations rely on the JSON-LD vocabulary (contained in this registry) for interoperability. 
+    </p>
+
+    <p>
+      Due to the nature of decentralization, JSON documents that contain terms 
+      not defined in a JSON-LD impossible to prevent.
+      Managing the validity / security considerations associated with unknown / undefined 
+      properties in the DID Document is the responsibility of the DID Method.
+    </p>
+
+    <p class="advisement">
+      Allowing unknown / undefined properties to be present in data structures that may be derived 
+      from untrusted user input has serious security implications.  
+      Please review <a href="https://cheatsheetseries.owasp.org/cheatsheets/Input_Validation_Cheat_Sheet.html">
+        OWASP Input Validation Cheat Sheet
+      </a> and the security and privacy section of registered DID Methods for additional details.
+    </p>
+
+    <p>
+      DID Methods and DID Parameters need only be registered via specification URLs.
```

Comment:
Grammar here is strange, don't quite know what you're saying. Seems like you're saying "The only thing you need to register a DID Method and DID Parameter" is a specification, but that's not true, it's not the only thing, because we outline a bunch of rules in DID Core about what that spec has to be, which you say in the very next sentence.

Reply:
agreed... the problem is that registration of did methods and params is very different from properties and values.... I was trying to say:

DID Methods and params are registered when by URL.

DID Properties and Values are registered via JSON-LD context defintion and URL.

Reply:
we might consider moving the registration sections into each sub section, so we don't need to address them all at once.

Reply:
removed sentence.